metricbeat/module/traefik/health/_meta/fields.yml

```diff
@@ -0,0 +1,45 @@
+- name: health
+  type: group
+  description: >
+    Metrics obtained from Traefik's health API endpoint
+  fields:
+    - name: uptime
+      type: group
+      description: >
+        Uptime of Traefik instance
+      fields:
+        - name: sec
+          type: int
+          description: >
+            Uptime in seconds
+    - name: response
+      type: group
+      description: >
+        Response metrics
+      fields:
+        - name: count
+          type: int
+          description: >
+            Number of responses
+        - name: avg_time
+          type: group
+          description: >
+            Average response time
+          fields:
+            - name: sec
+              type: float
```

Comment:
As discussed with the other sec type I struggle that we have a `sec` value that is float. I understand that for the average it makes more sense but in this case I would rather use a smaller unit.

Reply:
Makes sense, will use milliseconds and int here.

Reply:
For proxies performance monitoring it is useful to have sub-millisecond response times if available.